dandiapi/api/tasks.py

```diff
@@ -146,3 +146,51 @@ def validate_asset_metadata(version_id: int, asset_id: int) -> None:
     asset.status = Asset.Status.VALID
     asset.validation_error = ''
     asset.save()
+
+
+@shared_task
+@atomic
+def validate_version_metadata(version_id: int) -> None:
+    logger.info('Validating dandiset metadata for version %s', version_id)
+    version = Version.objects.get(id=version_id)
+
+    # Begin the validation process so no other validation tasks will run simultaneously
```

Comment:
Is this true? I don't see a check anywhere against the current state of the version validation.

Reply:
It is not, I will remove it.